package.json

```diff
@@ -41,6 +41,7 @@
     "@spectrum-web-components/action-button": "0.43.0",
     "@spectrum-web-components/action-group": "0.43.0",
     "@spectrum-web-components/action-menu": "0.43.0",
+    "@spectrum-web-components/badge": "^0.48.0",
```

Comment:
We want to keep all versions of SWC the same to avoid conflict issues that can occur with SWC when mixing versions. Can also remove the `^`.

Reply:
fixed